src/modules/dashboard/components/ActionsPageFeed/ActionsPageFeedItemWithIPFS/ActionsPageFeedItemWithIPFS.tsx

```diff
@@ -47,9 +52,21 @@ const ActionsPageFeedItemWithIPFS = ({
     if (!annotation || !ipfsDataJSON) {
       return undefined;
     }
-    const annotationObject = JSON.parse(ipfsDataJSON);
-    if (annotationObject && annotationObject.annotationMessage) {
-      return annotationObject.annotationMessage;
+
+    const metadataVersion = getEventMetadataVersion(ipfsDataJSON);
+    if (metadataVersion === 1) {
+      /*
+       * original metadata format
+       */
+      const annotationObject = JSON.parse(ipfsDataJSON);
+      if (annotationObject && annotationObject.annotationMessage) {
```

Comment:
I think `annotationObject` here is unnecessary because if `annotationObject.annotationMessage` is defined then `annotationObject` is by definition defined.

Reply:
Agree